modules/developer-lightspeed/proc-installing-and-configuring-lightspeed.adoc

```diff
@@ -134,385 +122,255 @@ data:
-              - name: LightspeedIcon
-                module: LightspeedPlugin
-                importName: LightspeedIcon
-            dynamicRoutes:
-              - path: /lightspeed
-                importName: LightspeedPage
-                module: LightspeedPlugin
-                menuItem:
-                  icon: LightspeedIcon
-                  text: Lightspeed
-  - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-lightspeed-backend:bs_1.39.1__0.5.7!red-hat-developer-hub-backstage-plugin-lightspeed-backend
-    disabled: false
-    pluginConfig:
-      lightspeed:
-        # REQUIRED: Configure LLM servers with OpenAI API compatibility
-        servers:
-          - id: ${LLM_SERVER_ID}
-            url: ${LLM_SERVER_URL}
-            token: ${LLM_SERVER_TOKEN}
-
-        # OPTIONAL: Port for lightspeed service (default: 8080)
-        # servicePort: ${LIGHTSPEED_SERVICE_PORT}
-----
-
-. Update your deployment configuration based on your installation method:
-.. For an Operator-installed {product-very-short} instance, update your {product-custom-resource-type} custom resource (CR).
-... In the `spec.application.appConfig.configMaps` section, add the {ls-short} custom app configuration as shown in the following example:
+. Update the dynamic plugins ConfigMap: Add the {ls-short} plugin image to your existing dynamic plugins ConfigMap (`dynamic-plugins-rhdh.yaml`).
 +
 [source,yaml]
 ----
-      appConfig:
-        configMaps:
-          - name: lightspeed-app-config
-        mountPath: /opt/app-root/src
+includes:
+- dynamic-plugins.default.yaml
+plugins:
+- package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-lightspeed:next__1.0.1!red-hat-developer-hub-backstage-plugin-lightspeed
+  disabled: false
+  pluginConfig:
+    lightspeed:
+      # OPTIONAL: Custom users prompts displayed to users
+      prompts:
+        - title: 'Getting Started with Red Hat Developer Hub'
+          message: Can you guide me through the first steps to start using Developer Hub as a developer, like exploring the Software Catalog and adding my service?
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-lightspeed:
+        appIcons:
+          - name: LightspeedIcon
+            module: LightspeedPlugin
+            importName: LightspeedIcon
+        dynamicRoutes:
+          - path: /lightspeed
+            importName: LightspeedPage
+            module: LightspeedPlugin
+            menuItem:
+              icon: LightspeedIcon
+              text: Lightspeed
+- package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-lightspeed-backend:next__1.0.1!red-hat-developer-hub-backstage-plugin-lightspeed-backend
+  disabled: false
+  pluginConfig:
+    lightspeed:
+      # OPTIONAL: Port for lightspeed service (default: 8080)
+      # servicePort: ${LIGHTSPEED_SERVICE_PORT}
```

Comment:
we can remove this in favour of the `lightspeed-app-config` configmap as it already contains the full configuration there.


```suggestion
```